pepper-apis/src/main/java/org/broadinstitute/ddp/route/GetStudyStatisticsRoute.java

```diff
@@ -0,0 +1,51 @@
+package org.broadinstitute.ddp.route;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.http.HttpStatus;
+import org.broadinstitute.ddp.constants.ErrorCodes;
+import org.broadinstitute.ddp.constants.RouteConstants;
+import org.broadinstitute.ddp.content.ContentStyle;
+import org.broadinstitute.ddp.content.I18nContentRenderer;
+import org.broadinstitute.ddp.db.TransactionWrapper;
+import org.broadinstitute.ddp.db.dao.JdbiUmbrellaStudy;
+import org.broadinstitute.ddp.db.dto.LanguageDto;
+import org.broadinstitute.ddp.db.dto.StudyDto;
+import org.broadinstitute.ddp.json.errors.ApiError;
+import org.broadinstitute.ddp.util.ResponseUtil;
+import org.broadinstitute.ddp.util.RouteUtil;
+import org.broadinstitute.ddp.util.StatisticsUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import spark.Response;
+import spark.Route;
+
+public class GetStudyStatisticsRoute implements Route {
+    private static final Logger LOG = LoggerFactory.getLogger(GetStudyStatisticsRoute.class);
+    private final I18nContentRenderer renderer;
+
+    public GetStudyStatisticsRoute(I18nContentRenderer renderer) {
+        this.renderer = renderer;
+    }
+
+    @Override
+    public Object handle(Request request, Response response) {
+        String studyGuid = request.params(RouteConstants.PathParam.STUDY_GUID);
+        ContentStyle style = RouteUtil.parseContentStyleHeaderOrHalt(request, response, ContentStyle.STANDARD);
+        LanguageDto preferredUserLanguage = RouteUtil.getUserLanguage(request);
+        if (StringUtils.isBlank(studyGuid)) {
+            LOG.warn("Study GUID is blank");
+            ResponseUtil.halt400ErrorResponse(response, ErrorCodes.MISSING_STUDY_GUID);
```

Comment:
If study guid is missing, it won't match the URL and this code won't run. I recommend removing this redundant block of code.

Reply:
Removed redundant piece of code